docs/content/docs/testing/telemetry.md

```diff
@@ -110,6 +110,8 @@ public sendTelemetryEvent(event: ITelemetryGenericEvent, error?: any) {
 Like demonstrated here, it is imperative to ensure `send()` is ultimately called at the end of custom properties for the
 information to be piped to the container's telemetry system and sends the telemetry event.
 
+Another point worth noting is that with the customized logics, it is a great place where you can send these events to your external telemetry system to have them logged.
```

Comment:
This is where I'm pointing out where in the implementation the developer would send events to their telemetry endpoint. Not sure if this is enough or need more elaboration.

Reply:
this callout should be below, where the doc covers a custom itelenetrybaselogger implementation.  when you put it here it's not clear what "it" in the callout refers to

Reply:
I was referring to having it in the `send()`, but I agree having it in the bottom section makes more sense.